src/openrct2/rct1/S4Importer.cpp

```diff
@@ -1243,6 +1243,15 @@ class S4Importer final : public IParkImporter
             }
         }
 
+        VEHICLE_STATUS statusSrc = VEHICLE_STATUS_MOVING_TO_END_OF_STATION;
+
+        // This will need to be updated if we ever add more VEHICLE_STATUS enums.
```

Comment:
I would just remove this comment for now it will be addressed by the enum class change.